ShoppingCart.Tests/CartConfigurationTests.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Xunit;
+using System.Text;
+
+namespace ShoppingCart.Tests
+{
+    public class CartConfigurationTests
+    {
+        [Theory]
+        [InlineData("iPhone")]
+        [InlineData("trousers")]
+
+        public void Check_discounts_for_individual_products(string ProductName)
+        {
+            Assert.Equal(10, CartConfiguration.GetDiscountPercentage(ProductName));
```

Comment:
GetDiscountPercentage method should not belong to CartConfiguration class as it is not responsibility of it to set or get discounts of product.